src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private static final int MAX_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
As mentioned here: https://github.com/mate-academy/jv-lottery/pull/1441#discussion_r1549187968
`ColorSupplier` should also be a constant. Try to check your code for objects that could be initialized once instead of being created every time a method is called.